content/runbooks/postgresql/PostgreSQLReplicationSlotStorage.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Replication slot storage
+---
+
+# PostgreSQLReplicationSlotStorage
```

Comment:
```suggestion
# PostgreSQLReplicationSlotStorageCloseToMax
```

Reply:
As discussed, I named it `PostgreSQLReplicationSlotStorageLimit`